### DISCUSSION TITLE
No Good first issues

### DISCUSSION THREAD

**Discussion by Mihan786Chistie** [juspay/hyperswitch#5799] [General]
I noticed there are no good first issues for beginners to start contributing. Any chance we can throw in some good first issues to help get people started? What do you all think?"

**Comment by neerajkumar-n** [comment]
@Mihan786Chistie - We have added plenty, can you take a look at them here: https://github.com/juspay/hyperswitch/labels/good%20first%20issue

**Comment by Mihan786Chistie** [comment]
Ohk thanks for thhe update @neerajkumar-n
